src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -328,13 +328,12 @@ static void usage(void)
     debugs(MY_DEBUG_SECTION, DBG_CRITICAL, "Wrong usage. Please reconfigure in squid.conf.");
 
     std::cerr <<
-              "Usage: " << program_name << " [-d] [-b dbpath] [-p pauselen] [-h] configfile\n"
-              "	-d            enable debugging output\n"
-              "	-l logfile    log messages to logfile\n"
-              "	-b dbpath     Path where persistent session database will be kept\n"
-              "	              If option is not used, then " DEFAULT_QUOTA_DB " will be used.\n"
-              "	-p pauselen   length in seconds to describe a pause between 2 requests.\n"
-              "	-h            show show command line help.\n"
+              "Usage: " << program_name << " [-d] [-b dbpath] [-p pauselen] [-h] configfile\n" <<
+              "	-d level      enable debugging output at level\n" <<
```

Comment:
How about
```
set debug level to specified, overriding any level set by squid for <MY_DEBUG_SECTION>
```
?

Reply:
Disclaimer: I do not know what problem you are trying to solve by altering my suggestion.

In general, I believe my suggestion is better because it hints at where the defaults are coming from (as you know, that squid.conf connection is rather obscure/surprising even for some core developers!) and because it documents the default value.

Please do not interpret the above as implying that my suggestion is problem-free. I am sure it can be improved. I just do not know how.

If you go with your suggestion, please capitalize "Squid".

Reply:
I think just removing the "overwriting debug_options" words from Alex suggestion is good. Since `debug_options` does not have any effect on helper outputs.

Reply:
Does it not?
debug_options is exported to the SQUID_DEBUG environment variable in ipc.cc:PutEnvironment, and then read by any helper using debugs() in src/debug.cc:Debug::NameThisHelper , as shown to me in [a previous pr](https://github.com/squid-cache/squid/pull/1847#discussion_r1712690746)

My proposal is meant to highlight this fact to administrators

Reply:
Ok, here's a new attempt that aims to hit correctness and clarity of obscure cases.
Does it look good (enough)?